server/lib/scraper.js

```diff
@@ -0,0 +1,107 @@
+const cheerio = require('cheerio');
+const request = require('request-promise');
+const parseHtml = require('../classes/Machine').parse;
+
+/**
+ * This function scrapes a list of all of the available laundry rooms. It returns an array of objects
+ * specifying both the name and full url of each room
+ */
+async function scrapeLocations(redis) {
```

Comment:
Consider changing name to "scrapeLocationNames" Or "LocationTitles" or similar. Is unclear at first if this meant scraping all info for a location or just the name.